src/editor/editorinteractive.cc

```diff
@@ -1077,7 +1077,7 @@ void EditorInteractive::do_run_editor(const EditorInteractive::Init init,
 		Notifications::publish(UI::NoteLoadingMessage(format(_("Loading map “%s”…"), filename)));
 		eia.load(filename);
 
-		egbase.postload_addons(true);
+		egbase.postload_addons();
```

Comment:
as we are postloading the addons in the egbase.postload() now do we need them to be loaded here as well.
Won't they be loaded twice

Reply:
`postload_addons` has a safeguard so repeated calls are ignored, however removed it